app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  property :id,               Serial
+  property :email,            String, required: true
+  property :password_digest,  Text
+
+  validates_presence_of     :password_digest
+  validates_uniqueness_of   :email
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.